src/backend/distributed/commands/table.c

```diff
@@ -733,20 +733,40 @@ PreprocessAlterTableStmt(Node *node, const char *alterTableCommand,
 
 	/*
 	 * check whether we are dealing with a sequence or view here
-	 * if yes, it must be ALTER TABLE .. OWNER TO .. command
-	 * since this is the only ALTER command of a sequence or view that
-	 * passes through an AlterTableStmt
 	 */
 	char relKind = get_rel_relkind(leftRelationId);
 	if (relKind == RELKIND_SEQUENCE)
 	{
 		AlterTableStmt *stmtCopy = copyObject(alterTableStatement);
 		AlterTableStmtObjType_compat(stmtCopy) = OBJECT_SEQUENCE;
+		#if (PG_VERSION_NUM >= PG_VERSION_15)
```

Comment:
nit, I think it looks better if we do not indent directives. I guess both would work as long as citus_indent is happy.

```suggestion
#if (PG_VERSION_NUM >= PG_VERSION_15)
```